test/integration/scheduler/framework_test.go

```diff
@@ -1080,12 +1080,11 @@ func TestReservePluginUnreserve(t *testing.T) {
 					}
 				}
 			}
-
-			preBindPlugin.reset()
-			for _, pl := range reservePlugins {
-				pl.reset()
-			}
-			testutils.CleanupPods(testCtx.ClientSet, t, []*v1.Pod{pod})
+			// preBindPlugin.reset()
```

Comment:
Is this comment intentional?

Reply:
Should probably be removed since each set of plugins is used only once.

Reply:
Yes, reset() are not necessary as each subtest is using its own plugin.